internal/search/alert.go

```diff
@@ -140,7 +140,7 @@ func AlertForStructuralSearchNotSet(queryString string) *Alert {
 	return &Alert{
 		PrometheusType: "structural_search_not_set",
 		Title:          "No results",
-		Description:    "It looks like you may have meant to run a structural search, but it is not toggled.",
+		Description:    "It looks like you may have meant to run a structural search, but the patterntype is not specified or it is not toggled.",
```

Comment:
```suggestion
		Description:    "It looks like you're trying to run a structural search, but it is not enabled using the patterntype keyword or UI toggle.",
```

Copied the same suggestion to the other lines 😄

Reply:
thanks for catching that, did this too late in my day